Projects/Modules/NetworkModule/Sources/Request/Encoding/ParamterJSONEncoder.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public struct ParamterJSONEncoder: RequestDataEncodable {
+    
+    func encode(request: inout URLRequest, with parameters: Parameters) throws {
```

Comment:
inout인 이유가 따로 있나요? 반환 타입이 있는 형태는 어떠신가요?
```suggestion
    func encode(request: inout URLRequest, with parameters: Parameters) throws -> URLRequest
```